config/environments/production.rb

```diff
@@ -58,7 +58,7 @@
   # config.cache_store = :mem_cache_store
 
   # Use a real queuing backend for Active Job (and separate queues per environment)
-  # config.active_job.queue_adapter     = :resque
+  config.active_job.queue_adapter     = :sidekiq
```

Comment:
We should probably configure the number of sidekiq workers in our deploy/prod.rb and start small